YOLOX/README.md

```diff
@@ -0,0 +1,238 @@
+|[YOLOX-l](./exps/default/yolox_l.py)    |640  |50.0  |14.5 |54.2| 155.6 | [DagsHub](https://dagshub.com/arnavr.neo/YOLOX/src/main/model/standard-models/legacy-models/yolox_l.pth) |
+|[YOLOX-x](./exps/default/yolox_x.py)   |640  |**51.2**      | 17.3 |99.1 |281.9 | [DagsHub](https://dagshub.com/arnavr.neo/YOLOX/src/main/model/standard-models/legacy-models/yolox_x.pth) |
+|[YOLOX-Darknet53](./exps/default/yolov3.py)   |640  | 47.4      | 11.1 |63.7 | 185.3 | [DagsHub](https://dagshub.com/arnavr.neo/YOLOX/src/main/model/standard-models/legacy-models/yolox_darknet53.pth) |
+
+</details>
+
+#### Light Models.
+
+|Model |size |mAP<sup>val<br>0.5:0.95 | Params<br>(M) |FLOPs<br>(G)| weights |
+| ------        |:---:  |  :---:       |:---:     |:---:  | :---: |
+|[YOLOX-Nano](./exps/default/yolox_nano.py) |416  |25.8  | 0.91 |1.08 | [DagsHub](https://dagshub.com/arnavr.neo/YOLOX/src/main/model/light-models/yolox_nano.pth) |
+|[YOLOX-Tiny](./exps/default/yolox_tiny.py) |416  |32.8 | 5.06 |6.45 | [DagsHub](https://dagshub.com/arnavr.neo/YOLOX/src/main/model/light-models/yolox_tiny.pth) |
+
+
+<details>
+<summary>Legacy models</summary>
+
+|Model |size |mAP<sup>val<br>0.5:0.95 | Params<br>(M) |FLOPs<br>(G)| weights |
+| ------        |:---:  |  :---:       |:---:     |:---:  | :---: |
+|[YOLOX-Nano](./exps/default/yolox_nano.py) |416  |25.3  | 0.91 |1.08 | [DagsHub](https://dagshub.com/arnavr.neo/YOLOX/src/main/model/light-models/legacy-models/yolox_nano.pth) |
+|[YOLOX-Tiny](./exps/default/yolox_tiny.py) |416  |32.8 | 5.06 |6.45 | [DagsHub](https://dagshub.com/arnavr.neo/YOLOX/src/main/model/light-models/legacy-models/yolox_tiny_32dot8.pth) |
+
+</details>
+
+## Quick Start
+
+<details>
+<summary>Installation</summary>
+
+Step1. Install YOLOX from source.
+```shell
+git clone https://github.com/arnavrneo/YOLOX.git
+cd YOLOX
+pip3 install -v -e .  # or  python3 setup.py develop
+```
+
+</details>
+
+<details>
+<summary>Demo</summary>
+
+Step1. Download a pretrained model from the benchmark table.
+
+Step2. Use either -n or -f to specify your detector's config. For example:
+
+```shell
+python tools/demo.py image -n yolox-s -c model/standard-model/yolox_s.pth --path assets/dog.jpg --conf 0.25 --nms 0.45 --tsize 640 --save_result --device [cpu/gpu]
+```
+or
+```shell
+python tools/demo.py image -f exps/default/yolox_s.py -c model/standard-model/yolox_s.pth --path assets/dog.jpg --conf 0.25 --nms 0.45 --tsize 640 --save_result --device [cpu/gpu]
+```
+Demo for video:
+```shell
+python tools/demo.py video -n yolox-s -c /path/to/your/yolox_s.pth --path /path/to/your/video --conf 0.25 --nms 0.45 --tsize 640 --save_result --device [cpu/gpu]
+```
+
+
+</details>
+
+<details>
+<summary>Reproduce our results on COCO</summary>
+
+Step1. Prepare COCO dataset from [DagsHub COCO Dataset](https://dagshub.com/Simon/COCO-2017) using `dvc import` command <a href="https://dvc.org/doc/command-reference/import">(more here)</a>.
```

Comment:
I think I was unclear in my previous message, I apologize for that. Currently in the repo, if you go into the dataset folder, there are instructions that contradict this, so it would be confusing to users. What I suggest is that you actually run this in your project, so that the future users can just `dvc pull` and will get everything` instead of needing to DVC import it themselves.